### DISCUSSION TITLE
输入法问题 只能用自带的不能用其他的

### DISCUSSION THREAD

**Discussion by vip458** [tyron12233/CodeAssist#305] [Q&A]
只能用自带的不能用其他的 我的自带输入法不能打中文

**Comment by tqz123** [comment]
设置中把安全键盘禁用就可以了
![IMG_20220824_183730.jpg](https://user-images.githubusercontent.com/61103056/186398071-cc4cd3f4-1250-4e9a-8f43-d477beed1078.jpg)
